UnityProject/Assets/Scripts/GameModes/Traitor.cs

```diff
@@ -68,11 +76,12 @@ public override void StartRound()
 	protected override bool ShouldSpawnAntag(PlayerSpawnRequest spawnRequest)
 	{
 
-		for (int i = 0; i < TraitorAmount - 1; i++) {
+		for (int i = 1; i < TraitorAmount; i++) {
 			return !LoyalImplanted.Contains(spawnRequest.RequestedOccupation.JobType)
 					&& PlayerList.Instance.InGamePlayers.Count > 0;
 		}
 		return !LoyalImplanted.Contains(spawnRequest.RequestedOccupation.JobType)
+					&& AntagManager.Instance.AntagCount == 0
 					&& PlayerList.Instance.InGamePlayers.Count > 0;

```

Comment:
Why the for loop? Why not replace
```
for (int i = 1; i < TraitorAmount; i++) {
	return !LoyalImplanted.Contains(spawnRequest.RequestedOccupation.JobType)
			&& PlayerList.Instance.InGamePlayers.Count > 0;
}
return !LoyalImplanted.Contains(spawnRequest.RequestedOccupation.JobType)
			&& AntagManager.Instance.AntagCount == 0
			&& PlayerList.Instance.InGamePlayers.Count > 0;
```
with
```
return !LoyalImplanted.Contains(spawnRequest.RequestedOccupation.JobType)
			&& AntagManager.Instance.AntagCount <= TraitorAmount
			&& PlayerList.Instance.InGamePlayers.Count > 0;
```

Reply:
Hmm I didn't really take that into account